alainmateso/iFind
# Chore Create Default Users

#### What does this PR do?
Creates seeders for default Users and Posts

#### Description of Task to be completed?
Create default seeders so that tests can run and sign in on existing accounts


#### How should this be manually tested?

1. Clone the repository
2. Run the ` npm install ` command
3. Run the ` npm run test ` command
4. Manually check your Postgres __TEST__ database for the following:  

- seeded user accounts: 

a. John Doe (johndoe@gmail.com)
b. Jane Doe (janedoe@gmail.com)

- seeded posts:

a. Description: 
> This is a demo post one. This post does the awesome post things. No post is more awesomer than this.

b. Description: 
> This is a demo post two! This post does the awesome post things. No post is more awesomest than this.

#### Any background context you want to provide?

N /A

#### What are the relevant pivotal tracker stories?
##### #168752605

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
